routers/web/web.go

```diff
@@ -671,6 +671,7 @@ func registerRoutes(m *web.Route) {
 		})
 
 		m.Group("/monitor", func() {
+			m.Get("/audit_logs", admin.ViewAuditLogs)
```

Comment:
Since when do we uses `_` in URLs?
```suggestion
			m.Get("/audit-logs", admin.ViewAuditLogs)
```

Reply:
```
m.Any("/activate_email", auth.ActivateEmail)
m.Get("/recover_account", auth.ResetPasswd)
m.Get("/forgot_password", auth.ForgotPasswd)
...
```
We have `_` in urls.